### PR TITLE
Add pytest repeat and testmon plugins

### PR DESCRIPTION
**Pull request by cleder**

### CODE REVIEW

File: .github/workflows/sync-fork.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    # Step 2: run this sync action - specify the upstream repo, upstream branch to sync with, and target sync branch
+    - name: Pull upstream changes
+      id: sync
+      uses: aormsby/Fork-Sync-With-Upstream-action@master
+      with:
+        upstream_repository: augustogoulart/awesome-pytest
+        upstream_branch: master
+        git_pull_args: --unshallow
+        target_branch: master                       # optional
+        github_token: ${{ secrets.GITHUB_TOKEN }}   # optional, for accessing repos that require authentication
```
Comment (augustogoulart) [inline .github/workflows/sync-fork.yml:30]: @cleder, what’s this intended for?

Comment (cleder) [reply .github/workflows/sync-fork.yml:30]: This is to keep Forks automatically synchronized to upstream (your repo) https://dev.to/github/sync-forks-to-upstream-using-github-actions-gle

Comment (augustogoulart) [reply .github/workflows/sync-fork.yml:30]: It's an interesting idea, but I'm not sure if I like that.

I would rather delegate to fork-owners updating their repos, especially if they are building something out of upstream and don't intend to sync at all. 

Let's remove that for now and I'll merge the other changes. 

Thanks for the PR!

**Comment by cleder** [comment]
New PR #25

**Pull request closed without merging**